mvrx/src/main/kotlin/com/airbnb/mvrx/Mavericks.kt

```diff
@@ -26,31 +26,47 @@ object Mavericks {
      * A factory for creating a [MavericksViewModelConfig] for each ViewModel.
      *
      * You MUST provide an instance here before creating any viewmodels. You can do this when
-     * your application is created via the [install] helper.
+     * your application is created via the [initialize] helper.
      *
      * This allows you to specify whether Mavericks should run in debug mode or not. Additionally, it
      * allows custom state stores or execution behavior for the ViewModel, which can be helpful
      * for testing.
      */
-    var viewModelConfigFactory: MavericksViewModelConfigFactory? = null
+    var viewModelConfigFactory: MavericksViewModelConfigFactory
+        set(value) {
+            _viewModelConfigFactory = value
+        }
+        get() {
+            _viewModelConfigFactory
```

Comment:
I don't think let is necessary
```suggestion
          return  _viewModelConfigFactory ?: error(...
```